features/partnerhistorie/peildatum-periode-filtering.feature

```diff
@@ -0,0 +1,258 @@
+# language: nl
+
+@post-assert
+Functionaliteit: Selecteer het/de correcte (historische) huwelijken geregistreerde partnerschappen.
+
+Rule: Bij vragen van partnerhistorie op peildatum wordt de partner geleverd waarbij de Datum huwelijkssluiting/aangaan geregistreerd partnerschap op of voor de peildatum ligt én de eventuele Datum ontbinding huwelijk/geregistreerd partnerschap na de peildatum.
+
+    Abstract Scenario: Peildatum in actuele huwelijk/partnerschap
+      Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende huwelijken/partnerschappen in de registratie
+      | Categorie | Burgerservicenummer (01.20) | Datum aangaan (06.10) | Datum ontbinding (07.10) | 
+      | 5         | 555550002                   | 20171103              |                          | 
+      | 55        | 555550003                   | 20101022              | 20140315                 | 
```

Comment:
Dit is volgens mij niet de correcte manier om een ontbonden partner op te geven. Dit komt niet overeen met hoe het in de oude partner historie scenario's wordt opgegeven. Volgens mij moet het als volgt worden geschreven:
```
| Categorie | Burgerservicenummer (01.20) | Datum aangaan (06.10) | Datum ontbinding (07.10) |
| 5         | 555550003                   |                       | 20140315                 |
| 55        | 555550003                   | 20101022              |                          |
```
Dit betekent dat je als lezer van deze scenario kennis moet hebben van hoe een ontbonden partner in de GBA wordt geregistreerd. Aangezien dit geen @gba scenario is, zou ik hier niet voor kiezen